inject.js

```diff
@@ -123,6 +123,8 @@
       } else {
         tc.settings.speeds[target.src] = tc.settings.lastSpeed;
       }
+      target.playbackRate = tc.settings.speeds[target.src];
```

Comment:
Why is this necessary in context of this PR?

Reply:
This is just reorganization of line 155 moved up to here. It makes sense to have all speed initialization things together and all Listeners grouped together.

Reply:
With the big new mutation block it seemed even more out of place

Reply:
👍